pkg/query/fanin.go

```diff
@@ -4,6 +4,8 @@ import (
 	"sort"
 	"unsafe"
 
+	"errors"
+
```

Comment:
Where does this import come from? If it's legit it wouldn't have compiled before, no?